guide/glossary.md

```diff
@@ -251,19 +251,37 @@ A standard for multi-signature that uses Schnorr signatures. Previously, the mor
 
 ### Node
 
-Node refers to [software]({{ '/guide/getting-started/software/#nodes' | relative_url }}) that participates in the bitcoin network. It exchanges transaction data with other nodes, stores some or all of it, and verifies that transactions are valid. There is also dedicated [node hardware]({{ '/guide/getting-started/hardware/#hardware-wallets' | relative_url }}).
+A node refers to a Bitcoin client that participates in the Bitcoin network.
 
-#### Full node
+#### Full nodes
 
-A node that fully verifies all transactions and rules of the Bitcoin network.
+Full nodes are Bitcoin clients that fully verify and enforce the rules of the Bitcoin network. An example of a rule is that there will never be more than 21 million bitcoin.
 
-#### Pruned full node
+##### Archival full node
 
-A node that fully verifies all of the rules of the Bitcoin network with a much smaller storage capacity because it only stores transactions relevant to the users wallet.
+A full node that stores the entire Bitcoin block chain, as well as sends historical data to other nodes.
 
-#### Archival node
+##### Pruned full node
 
-A full node that has not only verified all data, but also stores and provides it to other nodes on the network.
+A full node that uses much lower storage capacity as it only stores a portion of recent blocks. 
+
+#### Light nodes
+
+Light nodes are Bitcoin clients that do not verify or enforce the rules of the Bitcoin network.
+
+##### Simplified payment verification (SPV) light node
+
+A light node that queries and downloads block headers from full nodes making it possible for users to verify transactions without running a full node. 
+
+- [Bitcoin Wiki SPV](https://en.bitcoinwiki.org/wiki/Simplified_Payment_Verification)
+- [Bitcoin whitepaper - section 8](https://bitcoin.org/bitcoin.pdf)
+
+##### Neutrino light node
+
+A light node that uses the Neutrino protocol. The Neutrino protocol uses compact block filters to query and download condensed representations of blocks that contain transaction data relevant to the user.
```

Comment:
Would also be good to state here why this is beneficial, but maybe that also has to wait for a dedicated node page.

Reply:
If we eventually have a nodes chapter I think this can be detailed more there. Glossary's should be very short, succinct definitions not explanations imo.